README.md

```diff
@@ -68,93 +68,132 @@ services:
-    image: hashicorp/http-echo:${version}
+    image: hashicorp/http-echo
     command: ["-text", "${text}"]
     ports:
       - ${port}:5678
 ```
 
-Finally you can test everything is working, by rendering the Compose file with the provided default values.
+You can test everything is working, by inspecting the application.
+
+```console
+$ docker app inspect
+hello 0.1.0
+
+Maintained by: yourusername
+
+A simple text server
 
+Service (1) Replicas Ports Image
+----------- -------- ----- -----
+hello       1        5678  hashicorp/http-echo
+
+Parameters (2) Value
+-------------- -----
+port           5678
+text           hello development
 ```
-$ docker-app render
+
+You can render the application to a Compose file with the provided default
+values.
+
+```console
+$ docker app render
 version: "3.2"
 services:
   hello:
     command:
     - -text
     - hello development
-    image: hashicorp/http-echo:latest
+    image: hashicorp/http-echo
     ports:
     - mode: ingress
       target: 5678
       published: 5678
       protocol: tcp
 ```
 
-You can then use that Compose file like any other. You could save it to disk or pipe it straight to `docker stack` or `docker-compose` to launch the application.
+You can then use that Compose file like any other. You could save it to disk or
+pipe it straight to `docker stack` or `docker-compose` to launch the
+application.
 
-```
-$ docker-app render | docker-compose -f - up
+```console
+$ docker app render | docker-compose -f - up
 ```
 
-This is where it gets interesting. We can override those parameters at runtime, using the `--set` option. Let's specify different option and run `render` again:
+This is where it gets interesting. We can override those parameters at runtime,
+using the `--set` option. Let's specify different option and run `render` again:
 
-```
-$ docker-app render --set version=0.2.3 --set port=4567 --set text="hello production"
+```console
+$ docker app render --set port=4567 --set text="hello production"
 version: "3.2"
 services:
   hello:
     command:
     - -text
     - hello production
-    image: hashicorp/http-echo:0.2.3
+    image: hashicorp/http-echo
     ports:
     - mode: ingress
       target: 5678
       published: 4567
       protocol: tcp
 ```
 
-If you prefer you can create a standalone configuration file to store those parameters. Let's create `prod.yml` with the following contents:
+If you prefer you can create a standalone configuration file to store those
+parameters. Let's create `prod.yml` with the following contents:
 
 ```yaml
-version: 0.2.3
 text: hello production
 port: 4567
 ```
 
 You can then run using that configuration file like so:
 
+```console
+$ docker app render --parameters-file prod.yml
 ```
-$ docker-app render -f prod.yml
+
+You can share your application package by pushing it to a container registry.
+
+```console
+$ docker app push --tag myrepo/hello:0.1.0
+```
+
+Others can then use your application by specifying the registry tag.
```

Comment:
Eventually add that every command can work with a registry tag (render/inspect/install/...) ?